teps/0138-decouple-api-and-feature-versioning.md

```diff
@@ -191,22 +191,17 @@ When all existing `alpha` and `beta` features have either been stabilized or rem
+| [Object Parameters and Results](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/pipelineruns.md#specifying-parameters)                                | beta            |                                                 |
+| [Remote Tasks](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/taskruns.md#remote-tasks) and [Remote Pipelines](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/pipelineruns.md#remote-pipelines) | beta            |                                                 |
+| [Isolated `Step` & `Sidecar` `Workspaces`](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/workspaces.md#isolated-workspaces)                       | beta            |                                                 |
+| [Matrix](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs//matrix.md)                                                                                 | beta           |                                                 |
+| [Task-level Resource Requirements](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/compute-resources.md#task-level-compute-resources-configuration)                                                          | beta            |                                                 |
+| [Hermetic Execution Mode](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/hermetic.md)                                                              | alpha           |                                                 |
+| [Windows Scripts](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/tasks.md#windows-scripts)                                                         | alpha           |                                                 |
+| [Debug](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/debug.md)                                                                                   | alpha           |                                                 |
+| [Step and Sidecar Overrides](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/taskruns.md#overriding-task-steps-and-sidecars)                        | alpha           |                                                 |
+| [Configure Default Resolver](https://github.com/tektoncd/pipeline/blob/f78bcff9665f717fcc96644e04ba17375d0bd9a8/docs/resolution.md#configuring-built-in-resolvers)                          | alpha           |                                                 |
```

Comment:
Thanks, I think the `cel-in-WhenExpression` is now implemented. I think it can be added here.

"Param Enum" is under development, I can add it to here when the implementation is completed

Reply:
The point of this table is to record the `alpha` and `beta` features that are going to be sunset once they become stable or get removed. I am not assuming this table to be changed frequently as new features would go under Per-feature flags.

I've updated the explanation accordingly to avoid the confusion. Thanks for pointing this out!

Reply:
Oh I see, my mistake. Thanks!

Reply:
/lgtm